### PR TITLE
Update servers_v6.json

### PR DESCRIPTION
**Pull request by unjown**
FIRST MODDED SERVER

i have taken polls, looked over other server people want modded server but their staff don't  so I have decided to open this for people to enjoy :smile:

### CODE REVIEW

**Comment by nichrosia** [comment]
:eyes:

**Comment by alexpvpmindustry** [comment]
looks good! solid server! about time to have modded server in the public list

**Comment by BlueWolf3682** [comment]
We don't have support for downloading mods from server. Also, braindustry is a scripted mod and does not work on iOS. 

Not disagreeing with this idea, but the game doesn't support.

**Comment by WilloIzCitron** [comment]
> We don't have support for downloading mods from server. Also, braindustry is a scripted mod and does not work on iOS.
> 
> Not disagreeing with this idea, but the game doesn't support.

hmm Anuke has added a Mod Browser, but its for IOS?

File: servers_v6.json [hunk 1]
```diff
@@ -90,5 +90,9 @@
   {
     "name": "Darkdustry",
     "address": ["mindurka.ru"]
+  },
+  {
+    "name": "Unjown",
```
Comment (WilloIzCitron) [inline servers_v6.json:95]:
```suggestion
    "name": "Unjown Server",
```

Ah yes rename Unjown to Unjown Server

Comment (Quezler) [reply servers_v6.json]: sounds a bit redundant, its already a server and all the other groups also don't have that suffix 🤔

**Comment by walksanatora** [comment]
> FIRST MODDED SERVER
> 
> i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy

not the first
see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)

**Comment by Quezler** [comment]
@walksanatora while that is technically true, i don't require players to join with any non hidden mods, its all just by messing with the serverside code, this pull on the other hand is likely a server that requires the players to download and install something before joining 😗

**Comment by unjown** [comment]
> > FIRST MODDED SERVER
> > i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy
> 
> not the first
> see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)

when i mean mod 

literally mod :smile: the one that adds more content

**Comment by Quezler** [comment]
i wonder how anuke would like to deal with those, possibly by putting them all in a single group called modded or something, or introduce another layer of category-ing ontop 🤔

**Comment by unjown** [comment]
> ng, or introduce another layer of category-ing ontop



> i wonder how anuke would like to deal with those, possibly by putting them all in a single group called modded or something, or introduce another layer of category-ing ontop

we have to wait for around 8hrs (its the time anuke usually starts merging/commiting  and releases new versions)

**Comment by sk7725** [comment]
> > > FIRST MODDED SERVER
> > > i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy
> > 
> > 
> > not the first
> > see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)
> 
> when i mean mod
> 
> literally mod 😄 the one that adds more content

Still not true, Omega Server has a BetaMindy modded server already

**Comment by RebornTrack970** [comment]
> > > > FIRST MODDED SERVER
> > > > i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy
> > > 
> > > 
> > > not the first
> > > see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)
> > 
> > 
> > when i mean mod
> > literally mod 😄 the one that adds more content
> 
> Still not true, Omega Server has a BetaMindy modded server already

it exists for about 3 weeks so...
i think braindustry server is older. anyways, it would be pog if mods could add servers to the list.

**Comment by RebornTrack970** [comment]
> i wonder how anuke would like to deal with those, possibly by putting them all in a single group called modded or something, or introduce another layer of category-ing ontop 🤔

maybe if mods could add servers themselfs, it wuld solve everything.

**Comment by DeltaNedas** [comment]
1. ios should not gimp other platforms
2. missing a comma buddy
3. where is routorio

**Comment by unjown** [comment]
> 1. ios should not gimp other platforms
> 2. missing a comma buddy
> 3. where is routorio

check latest commit added comma

**Comment by unjown** [comment]
> > > > FIRST MODDED SERVER
> > > > i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy
> > > 
> > > 
> > > not the first
> > > see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)
> > 
> > when i mean mod
> > 
> > literally mod 😄 the one that adds more content
> 
> Still not true, Omega Server has a BetaMindy modded server already

not in public

also braindustry one has been over 3 weeks in hub

**Comment by RebornTrack970** [comment]
> > > > > FIRST MODDED SERVER
> > > > > i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy
> > > > 
> > > > 
> > > > not the first
> > > > see v6.mindustry.nydus.app:6566 (which is currently bugged but use /readme)
> > > 
> > > 
> > > when i mean mod
> > > literally mod 😄 the one that adds more content
> > 
> > 
> > Still not true, Omega Server has a BetaMindy modded server already
> 
> not in public
> 
> also braindustry one has been over 3 weeks in hub

Still, unless modded-non modded server issues get fixed, no modded servers on the list... (probably)

**Comment by WilloIzCitron** [comment]
f

**Comment by unjown** [comment]
F

**Review by alexpvpmindustry — APPROVED** [review]
server looks good! 
image
look forward to modded PvP

**Comment by ChrisThePersonWithoutSanity** [comment]
poggers.

**Comment by unjown** [comment]
Anuken pls add :pray: :smile:

**Comment by RebornTrack970** [comment]
> Anuken pls add 🙏 😄

No begging, wait till he either rejects or merges.

**Comment by WilloIzCitron** [comment]
lol f

**Comment by BlueWolf3682** [comment]
not gonna lie, that image shows me some worrying spriting

ill check the mod later to make sure

**Comment by WilloIzCitron** [comment]
we takes so long on this pr and Anuke doesnt respond on this pr

**Comment by unjown** [comment]
> we takes so long on this pr and Anuke doesnt respond on this pr
for unknown reason anuke has not been merging that much these days

**Comment by RebornTrack970** [comment]
> > we takes so long on this pr and Anuke doesnt respond on this pr
> 
> for unknown reason anuke has not been merging that much these days

It took 8 days for my servers to get merged, he mostly waits until enough people accept/approve the pull.
Also, there are like 7 more server pulls on rn.

**Comment by RebornTrack970** [comment]
> FIRST MODDED SERVER
> 
> i have taken polls, looked over other server people want modded server but their staff don't so I have decided to open this for people to enjoy 😄

https://github.com/pixaxeofpixie/Braindustry-Mod/pull/36

**Comment by pixaxeofpixie** [comment]
the mod may have problems on the servers, especially in conjunction with other mods, because before that Braindustry was on the Pandorum server and there were various unpleasant things there, although they were not in the solo game, just if they are, I'm not even sure that I can fix them.

**Comment by unjown** [comment]
> the mod may have problems on the servers, especially in conjunction with other mods, because before that Braindustry was on the Pandorum server and there were various unpleasant things there, although they were not in the solo game, just if they are, I'm not even sure that I can fix them.

i run it solo only 
yes if played with other mods it has a chance of bugging

it seems ok to me havent encountered any bug on server

**Comment by unjown** [comment]
> > > we takes so long on this pr and Anuke doesnt respond on this pr
> > 
> > 
> > for unknown reason anuke has not been merging that much these days
> 
> It took 8 days for my servers to get merged, he mostly waits until enough people accept/approve the pull.
> Also, there are like 7 more server pulls on rn.

true , alex already approve waiting for more 
~~RTQ approve this~~

**Comment by RebornTrack970** [comment]
> > > > we takes so long on this pr and Anuke doesnt respond on this pr
> > > 
> > > 
> > > for unknown reason anuke has not been merging that much these days
> > 
> > 
> > It took 8 days for my servers to get merged, he mostly waits until enough people accept/approve the pull.
> > Also, there are like 7 more server pulls on rn.
> 
> true , alex already approve waiting for more
> ~RTQ approve this~

No, the pull i made to Braindustry already adds this server to the list IF the user has the mod installed.
Thus, making this pull basicaly useless.

**Comment by RebornTrack970** [comment]
> > > > we takes so long on this pr and Anuke doesnt respond on this pr
> > > 
> > > 
> > > for unknown reason anuke has not been merging that much these days
> > 
> > 
> > It took 8 days for my servers to get merged, he mostly waits until enough people accept/approve the pull.
> > Also, there are like 7 more server pulls on rn.
> 
> true , alex already approve waiting for more
> ~RTQ approve this~

No, the pull i made to Braindustry already adds this server to the list IF the user has the mod installed.
Thus, making this pull basicaly useless.

**Comment by unjown** [comment]
> No, the pull i made to Braindustry already adds this server to the list IF the user has the mod installed.
> Thus, making this pull basicaly useless.

i dont get?

**Comment by Anuken** [comment]
I don't like the idea of there being modded servers on the public list, as no mechanism to automatically download mods exists. Including modded servers *in the mod* seems like a better solution.

**Comment by WilloIzCitron** [comment]
oh modded server itsnt requested for added on community list?

**Pull request closed without merging**